src/Screens/DiveLogsForms/index.tsx

```diff
@@ -135,11 +135,12 @@ const SimpleDiveLogsForms: FunctionComponent<
     try {
       setFormSubmitting(true);
 
-      let arrangedValues = {
+      let arrangedValues: InitialValues = {
         ...values,
         beach_id: values.location?.beach_id,
         dive_shop_id: values.dive_shop?.shop_id,
         is_private: values.privacy === t('DIVE_LOG_PRIVATE').toLowerCase(),
+        date_dived: new Date().toJSON(),
```

Comment:
we may need to ensure that this date is in a specific format. i'll double check on the frontend code and send you an example

Reply:
Sure. This one reads like `1975-08-19T23:15:30.000Z`. Basically a string representation of the `Date` object..

Reply:
okay i checked and this is the correct format